articles/migrations/index.md

```diff
@@ -71,22 +73,55 @@ If you are currently using [/delegation](/api/authentication#delegation) to prov
 
 If you have any questions, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
 
-### Deprecating the Usage of ID Tokens on the Auth0 Management API
+### Deprecating the usage of ID Tokens on the Auth0 Management API
+
+| Severity | Grace Period Start | Mandatory Opt-In|
+| --- | --- | --- |
+| Medium | 2017-12-21 |  2018-06-01 |
+
+We are deprecating the usage of [ID Tokens](/tokens/id-token) as credentials when calling the [Management API](/api/management/v2#!/Users/post_identities). This was used by the [/users](/api/management/v2#!/Users/get_users_by_id) and [/device-credentials](/api/management/v2#!/Device_Credentials/get_device_credentials) endpoints. 
+
+The affected endpoints are:
+
+- [GET /api/v2/users/{id}](/api/management/v2#!/Users/get_users_by_id)
+- [GET /api/v2/users/{id}/enrollments](/api/management/v2#!/Users/get_enrollments)
+- [PATCH /api/v2/users/{id}](/api/management/v2#!/Users/patch_users_by_id)
+- [DELETE /api/v2/users/{id}/multifactor/{provider}](/api/management/v2#!/Users/delete_multifactor_by_provider)
+- [POST /api/v2/device-credentials](/api/management/v2#!/Device_Credentials/post_device_credentials)
+- [DELETE /api/v2/device-credentials/{id}](/api/management/v2#!/Device_Credentials/delete_device_credentials_by_id)
+- [POST/api/v2/users/{id}/identities](/api/management/v2#!/Users/post_identities) (used for [Account Linking](/link-accounts))
+- [DELETE /api/v2/users/{id}/identities/{provider}/{user_id}](/api/management/v2#!/Users/delete_provider_by_user_id) (used for [Account Linking](/link-accounts))
+
+These endpoints will now accept regular [Access Tokens](/access-token). This functionality is available now.
+
+To get a valid Access Token for these endpoints during authorization, you have to set the **audience** parameter to `https://${account.namespace}/api/v2/` and the **scope** parameter to the scopes required by each endpoint.
+
+Note that the following scopes are also required by some endpoints:
+
+- `read:current_user` (required by: [GET /api/v2/users/{id}](/api/management/v2#!/Users/get_users_by_id), [GET /api/v2/users/{id}/enrollments](/api/management/v2#!/Users/get_enrollments))
+- `update:current_user_identities` (required by: [POST/api/v2/users/{id}/identities](/api/management/v2#!/Users/post_identities), [DELETE /api/v2/users/{id}/identities/{provider}/{user_id}](/api/management/v2#!/Users/delete_provider_by_user_id))
+- `update:current_user_metadata` (required by: [PATCH /api/v2/users/{id}](/api/management/v2#!/Users/patch_users_by_id))
```

Comment:
@cabralmartin Is this required to update *any* of the user info, or specifically the `user_metadata`?